src/terraboot/core.clj

```diff
@@ -6,12 +6,8 @@
             [clojure.pprint :refer [pprint]]
             [clojure.set :as set]))
 
-(def account-id "165664414043")
-(def default-sgs ["allow_ssh" "allow_outbound"])
 
-(def ubuntu "ami-9b9c86f7")
-(def current-coreos-ami "ami-1807e377")
-(def ec2-ami "ami-bc5b48d0")
+(def default-sgs ["allow_ssh" "allow_outbound"])
```

Comment:
@elisehuard not sure how I feel about the :refer :all rather than something like :as t